www/doc/1.7/report.html

```diff
@@ -20,143 +22,176 @@ <h1>Herbie reports</h1>
+    Third, under <em>Local Percentage Accuracy vs x</em>, a graph of floating-point accuracy comparing input vs Herbie's
+    <em>Most accurate alternative</em>. This is helpful for understanding the
+    sorts of inputs Herbie is improving accuracy on. Sometimes, Herbie improved
     accuracy on some inputs at the cost of accuracy on other inputs
     that you care more about. You can add
     a <a href="input.html#preconditions">precondition</a> to restrict
     Herbie to certain inputs in those cases.
   </p>
 
   <p>
-    These graphs show the error of the input program with a red line,
-    and the error of the output program with a blue line. Both can be
-    toggled. If the expression has multiple variables, the variable
-    picker on the bottom left selects which variable is placed on the
+    These graphs show the accuracy of the input program with a red line,
+    and the accuracy of the <em>Most accurate Alternative</em> program with a
+    blue line. Both can be toggled. If the expression has multiple variables,
+    the variable picker on the top left selects which variable is placed on the
     horizontal axis. If Herbie decided to insert an <code>if</code>
     statement into the program, the locations of those <code>if</code>
     statements will be marked with vertical bars.
   </p>
 
   <figure>
-    <img width="100%" src="report-error.png" />
-    <figcaption>An error graph from a Herbie report. Note the variable
-    selector (<code>x</code> is selected) and the toggles for the
-    input and output program (both are toggled on).</figcaption>
+    <img width="100%" src="report-accuracy.png" />
+    <figcaption>An accuracy graph from a Herbie report. Note the variable
+      selector (<code>x</code> is selected) and the toggles for the
+      input and output program (both are toggled on).</figcaption>
   </figure>
 
-  <h2 id="bogosity">Bogosity</h2>
-  <p>
-  Bogosity is a meter to glance at the validity of a program. Points 
-  sampled by Herbie will fall into one of the fallowing categories.
-  <ul>
-     <li>Valid - Correctly rounded</li>
-     <li>Invalid - Proven to be invalid sqrt(-1)</li>
-     <li>Infinite - Proven to have very large output</li>
-     <li>Unsamplable - Interval Analysis didn't converge</li>
-  </ul>
-  </p>
-
-  <h2 id="try-it">Interactive inputs</h2>
+  <h2 id="cost-accuracy">Accuracy vs Speed</h2>
 
   <p>
-    Fourth, a interactive form where you can the output of both your
-    and Herbie's programs. on inputs of your choice Enter argument
-    values on the left, and the input and output programs will be
-    evaluated on those arguments and the results printed on the right.
+    Fourth, a Pareto curve where up and to the right is best. The input program
+    is marked with a red box and alternatives with blue dots. Along the right
+    side you have a list of alternatives with there accuracy and speedup listed relative to the input. Stats considered
```

Comment:
It's not clear enough from this description that the dots and the list of alternatives are listing the same data.